webapp/src/main/java/life/qbic/views/login/LoginHandler.java

```diff
@@ -57,24 +57,36 @@ public void handle(LoginLayout loginView) {
   }
 
   private void initFields() {
-    clearMessages();
-  }
-
-  private void clearMessages() {
-    registeredLoginView.clearErrors();
-    registeredLoginView.clearInformation();
+    clearNotifications();
   }
 
   private void showInvalidCredentialsError() {
-    registeredLoginView.showError(INCORRECT_USERNAME_OR_PASSWORD);
+    showError(INCORRECT_USERNAME_OR_PASSWORD);
   }
 
   private void showEmailConfirmationInformation() {
-    registeredLoginView.showInformation(EMAIL_CONFIRMATION_SUCCESS);
+    showInformation(EMAIL_CONFIRMATION_SUCCESS);
   }
 
   private void showEmailConfirmationReminder() {
-    registeredLoginView.showInformation(EMAIL_CONFIRMATION_REMINDER);
+    showInformation(EMAIL_CONFIRMATION_REMINDER);
+  }
+
+  public void clearNotifications() {
+    registeredLoginView.notificationLayout.setVisible(false);
+    registeredLoginView.notificationLayout.removeAll();
```

Comment:
If you remove all you can also leave the layout visible, right?

Reply:
Since the layout itself does not have margins and paddings i think you're right